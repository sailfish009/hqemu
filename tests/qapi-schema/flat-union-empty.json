# FIXME - flat unions should not be empty
{ 'enum': 'Empty', 'data': [ ] }
{ 'struct': 'Base', 'data': { 'type': 'Empty' } }
{ 'union': 'Union', 'base': 'Base', 'discriminator': 'type', 'data': { } }
