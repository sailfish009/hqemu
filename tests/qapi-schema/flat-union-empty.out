object :empty
object Base
    member type: Empty optional=False
enum Empty []
object Union
    base Base
    tag type
